www/js/settings.json

```diff
@@ -92,8 +92,29 @@
               "long": "Long",
               "extralong": "Extra Long"
             }
+          },
+          "vishraam-options": {
+            "title": "Vishraam Options",
+            "options": {
+              "colored-words": "Colored Words",
+              "gradient-bg": "Gradient Background"
+            }
+          },
+          "vishraam-source": {
+            "title": "Vishraam Source",
+            "options": {
+              "sttm2": "STTM 2",
+              "igurbani": "iGurbani",
```

Comment:
need to give some credit here (also in the help)?

Reply:
I'm assuming you mean credit to the other orgs for the other vishraam sources. Yes I will update this info in the help.